.travis.yml

```diff
@@ -14,34 +18,46 @@ env:
 matrix:
   include:
     # current stable release w/ multisite
-    - php: "5.6"
-      env: WP_VERSION=master WP_MULTISITE=1
-    - php: "5.6"
-      env: WP_VERSION=4.5 WP_MULTISITE=0
-    - php: "5.6"
-      env: WP_VERSION=4.5 WP_MULTISITE=1
-    - php: "5.6"
-      env: WP_VERSION=4.4 WP_MULTISITE=0
-    - php: "5.5"
-      env: WP_VERSION=master WP_MULTISITE=0    
+    - php: "5.4"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
     - php: "5.5"
-      env: WP_VERSION=4.5 WP_MULTISITE=0
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "5.6"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "7.0"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "7.1"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "5.4"
+      env: WP_VERSION=4.8 WP_MULTISITE=1
     - php: "5.5"
-      env: WP_VERSION=4.4 WP_MULTISITE=0    
-    - php: "5.2"
```

Comment:
Totally get the idea behind dropping PHP 5.2 and am all-in for doing so. Just, could we adjust the comment on line 14 to match this change?

Reply:
I'm sorry but I don't understand what's the comment I should edit. Is it the one here? https://github.com/Automattic/Edit-Flow/blob/master/.travis.yml#L16

Reply:
I should had provided link right away :) I have been using the line numbers in your brach/fork: https://github.com/raduconst/Edit-Flow/blob/69455a92b45a22727e61f30e2da7ed3be3fa1bd6/.travis.yml#L14

> on minimum PHP 5.2

which is no longer true. We should make sure the comments in the file are accurate. Comments and code should always be consistent and updated along each other, unless future reader can't tell whether the comment is wrong, when clearly saying the opposite of what the code is doing, or whether that's a bug.

With all the changes in the test matrix, there might be other comments which need to be updated, even if only slightly.

Reply:
You are right. I don't know how could I miss that. Yes, this is very important to be changed and I think that I can do that based on what we discussed here.